openmetadata-ui/src/main/resources/ui/src/styles/components/table.less

```diff
@@ -9,14 +9,21 @@
   .ant-table-bordered {
     table {
       border: 1px solid #dde3ea;
-      box-shadow: 1px 1px 3px 0px rgba(0, 0, 0, 0.12);
     }
     .ant-table-container > .ant-table-content > table > tbody > tr > td {
       border-right: none;
     }
   }
 }
 
+.table-shadow {
+  .ant-table-bordered {
+    table {
+      box-shadow: 1px 1px 3px 0px rgba(0, 0, 0, 0.12);
+    }
+  }
+}
+
```

Comment:
Table has shadow already ASAIK

Reply:
Table shadows were applied globally, I moved it to a global class 'table-shadow' to give shadows only to the tables we want